src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -25,13 +28,52 @@ export class BootstrapService {
     try {
       console.info('Bootstrapping Ecoverse...');
       await this.ensureEcoverseSingleton();
-      await this.ensureAdminRole();
+      await this.bootstrapProfiles();
       await this.validateAccountManagementSetup();
     } catch (error) {
       console.log(error);
     }
   }
 
+  async bootstrapProfiles() {
+    const bootstrapFilePath = this.configService.get<IServiceConfig>('service')
+      ?.authorisationBootstrapPath as string;
+
+    const bootstrapData = await import(bootstrapFilePath);
+    await this.createGroupProfiles(
+      RestrictedGroupNames.Admins,
+      await bootstrapData[RestrictedGroupNames.Admins]
+    );
+    await this.createGroupProfiles(
+      RestrictedGroupNames.GlobalAdmins,
+      await bootstrapData[RestrictedGroupNames.GlobalAdmins]
+    );
+    await this.createGroupProfiles(
+      RestrictedGroupNames.CommunityAdmins,
+      await bootstrapData[RestrictedGroupNames.CommunityAdmins]
+    );
+  }
+
+  async createGroupProfiles(groupName: string, emails: string[]) {
+    try {
+      emails.forEach(async email => {
+        const userInput = new UserInput();
+        userInput.email = email;
+        userInput.name = 'Imported User';
+        //env variable AUTHENTICATION_ENABLED=false. Otherwise this won't work.
```

Comment:
Worth putting a check in that accounts are disabled? There is a function for that already via the AccountService.

Reply:
Fixed in d6f43ad5ea6b78b4b79d6889962971a5eae009eb